common/components/ui/Dropdown.tsx

```diff
@@ -3,6 +3,7 @@ import classnames from 'classnames';
 import DropdownShell from './DropdownShell';
 
 interface Props<T> {
+  className?: string;
```

Comment:
Using `className` in custom props is kind of a faux paus, most components don't do it. If you want to alter the way it looks, you typically pass a more specific prop, like `color` or `size`. If you want to alter the way it's positioned or spaced, you typically wrap a `<div>` around the component and adjust the sizing or spacing of that instead.

Basically the problem is that if there's some CSS that the component relies on, i.e. negative margins or some positioning tricks, you might override it with your custom class. It makes reusable components a little hairier if they can get overridden like that.